app/cdap/components/FooterPanel/Components/common/RenderLabel/index.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Typography } from '@material-ui/core';
+import { grey } from '@material-ui/core/colors';
+import { IRenderLabelProps } from 'components/FooterPanel/Components/common/RenderLabel/types';
+import React from 'react';
+import styled from 'styled-components';
+
+const SimpleLabel = styled(Typography)`
+  &&& {
+    margin-left: 2px;
+    margin-right: 5px;
+    margin-top: 2px;
+  }
+`;
+
+const OutlinedLabel = styled(Typography)`
+  &&& {
+    background-color: ${grey[600]};
+    height: 21px;
+    width: 20px;
+    color: #ffffff;
```

Comment:
please be consistent with color usage - if you're doing ${grey[600]} then I also expect the same thing for instead of #ffffff

Reply:
#ffffff is not available in mui colors (https://v4.mui.com/customization/color/#color)